instrumentation/spring/spring-webflux-5.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/spring/webflux/server/DispatcherHandlerInstrumentation.java

```diff
@@ -41,41 +37,20 @@ public void transform(TypeTransformer transformer) {
         this.getClass().getName() + "$HandleAdvice");
   }
 
-  /**
-   * This is 'top level' advice for Webflux instrumentation. This handles creating and finishing
-   * Webflux span.
-   */
   @SuppressWarnings("unused")
   public static class HandleAdvice {
 
-    @Advice.OnMethodEnter(suppress = Throwable.class)
-    public static void methodEnter(
-        @Advice.Argument(0) ServerWebExchange exchange,
-        @Advice.Local("otelScope") Scope otelScope,
-        @Advice.Local("otelContext") Context otelContext) {
-
-      otelContext = tracer().startSpan("DispatcherHandler.handle", SpanKind.INTERNAL);
-      // Unfortunately Netty EventLoop is not instrumented well enough to attribute all work to the
-      // right things so we have to store the context in request itself.
-      exchange.getAttributes().put(AdviceUtils.CONTEXT_ATTRIBUTE, otelContext);
-
-      otelScope = otelContext.makeCurrent();
-    }
-
     @Advice.OnMethodExit(onThrowable = Throwable.class, suppress = Throwable.class)
     public static void methodExit(
         @Advice.Thrown Throwable throwable,
         @Advice.Argument(0) ServerWebExchange exchange,
-        @Advice.Return(readOnly = false) Mono<Void> mono,
-        @Advice.Local("otelScope") Scope otelScope,
-        @Advice.Local("otelContext") Context otelContext) {
-      if (throwable == null && mono != null) {
-        mono = AdviceUtils.setPublisherSpan(mono, otelContext);
-      } else if (throwable != null) {
-        AdviceUtils.finishSpanIfPresent(exchange, throwable);
+        @Advice.Return(readOnly = false) Mono<Void> mono) {
+      if (mono != null) {
+        // note: it seems like this code should go in HandleAdvice @OnMethodExit
```

Comment:
Do you mean Enter? This code seems to be in HandleAdvice.OnMethodExit right now

Reply:
oh no, two different `HandleAdvice` classes, I fixed the comment 👍